README.md

```
@@ -222,72 +222,78 @@ If `SUPPORT_AGENT_NICKNAMES` is not configured:
+**Metadata Structure**:
+```json
+{
+  "title": "Document Title",
+  "category": "bisq2|bisq1|general",
+  "type": "wiki",
+  "source_weight": 1.1,
+  "protocol": "bisq_easy|multisig_v1|all"
+}
+```
 
 ### 2. FAQ Data
```

markdownlint-cli2 (0.20.0)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)